### ISSUE TITLE
Error parsing create table statement with ROWID as column name

### ISSUE THREAD

**Issue by cwimberger** [alicebob/sqlittle#26]
I've got a sqlite file with the following table structure:

`CREATE TABLE handle ( ROWID INTEGER PRIMARY KEY AUTOINCREMENT UNIQUE, id TEXT NOT NULL, country TEXT, service TEXT NOT NULL, uncanonicalized_id TEXT, person_centric_id TEXT DEFAULT NULL, UNIQUE (id, service) )`

It seems like the first column is named 'ROWID' so the lexer returns a ROWID instead of a tIdentifier. 

The debug output looks like this:

```
char tok--1 in state-0
lex CREATE(57355)
char tok--1 in state-6
lex TABLE(57387)
char tok--1 in state-13
lex tBare(57392)
char tok--1 in state-11
reduce 6 in:
        state-11
char tok--1 in state-18
lex '('(40)
char tok--1 in state-22
lex ROWID(57384)
state-22 saw ROWID
error recovery pops state 22
error recovery pops state 18
error recovery pops state 13
error recovery pops state 6
error recovery pops state 0
panic: syntax error: unexpected ROWID, expecting tBare or tIdentifier
```

**Comment by alicebob** [comment]
On Sat, Feb 29, 2020 at 05:16:53AM -0800, Christoph Wimberger wrote:
> I've got a sqlite file with the following table structure:

Thanks for the bugreport! I'll have a look in a bit.

**Comment by alicebob** [comment]
Thanks, fixed in v1.3.1
